databases/week1/hyf_lesson1.sql

```diff
@@ -0,0 +1,52 @@
+--1-- Find out how many tasks are in the task table
+SELECT count(*)
+FROM task;
+
+--2-- Find out how many tasks in the task table do not have a valid due date
+SELECT count(*)
+FROM task
+WHERE due_date is NULL;
+
+--3-- Find all the tasks that are marked as done
+SELECT *
+from task
+where status_id = 3;
+
+--4-- Find all the tasks that are not marked as done
+SELECT *
+FROM task
+WHERE NOT status_id=3;
+
+--5-- Get all the tasks, sorted with the most recently created first
+SELECT *
+from task
+ORDER BY created ASC;
```

Comment:
This gives the oldest one, you want DESC instead of ASC

Reply:
I was missunderstand this. I thought its need to be the oldest one. I will update this 👍